system_metrics_collector/test/system_metrics_collector/test_periodic_measurement_node.cpp

```diff
@@ -171,34 +182,83 @@ TEST_F(PeriodicMeasurementTestFixure, TestStartAndStop) {
 }
 
 TEST_F(PeriodicMeasurementTestFixure, TestConstructorInputValidation) {
+  std::vector<rclcpp::Parameter> parameter_overrides;
+  std::vector<std::string> arguments;
+
+  // bad measurement period
+  parameter_overrides.push_back(rclcpp::Parameter(
```

Comment:
Is it better to have these tests separated?

Reply:
Done.